Source/JavaScriptCore/runtime/LiteralParser.h

```diff
@@ -151,25 +151,34 @@ class LiteralParser {
     
     JSValue tryLiteralParse()
     {
+        ASSERT(m_mode == StrictJSON);
+        ASSERT(reviverMode == JSONReviverMode::Disabled);
```

Comment:
Ditto.  Make this a `static_assert` on `reviverMode`.

Reply:
Ditto.